src/Emulator/Tabs/ConfigureTab/ConfigureTab.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from "react";
+import "./ConfigureTab.css";
+import DeviceContainer from "./DeviceContainer/DeviceContainer";
+
+// Sample responses from currently non-existent API
+// These represent the starting config for a lesson
+const routerConfig = [
+  {
+    name: "r1",
+    connections: ["s1", "s2"],
+  },
+];
+const switchConfig = [
+  {
+    name: "s1",
+    connections: ["r1", "h1", "h2", "h3"],
+  },
+  {
+    name: "s2",
+    connections: [],
+  },
+];
+const hostConfig = [
+  {
+    name: "h1",
+    connections: ["s1"],
+  },
+  {
+    name: "h2",
+    connections: ["s1"],
+  },
+  {
+    name: "h3",
+    connections: ["s1"],
+  },
+  {
+    name: "h4",
+    connections: [],
+  },
+];
+
+const ConfigureTab = ({ status }) => {
+  const [routers, setRouters] = useState(routerConfig);
+  const [switches, setSwitches] = useState(switchConfig);
+  const [hosts, setHosts] = useState(hostConfig);
+
+  const getNextNumber = (s) => parseInt(s.slice(1, s.length)) + 1;
```

Comment:
Flawlessly executed :100: You can also omit the second argument for `slice`, when not provided it goes to the end by default. Another function with similar functionality to `parseInt` you might see is `Number()`